starboard/build/config/starboard_target_type.gni

```diff
@@ -37,6 +41,17 @@ template("starboard_platform_target") {
     }
     if (target_type == "shared_library") {
       build_loader = false
+      install_target = false
```

Comment:
We need an install_target for this target still—is it generated some other way?

Reply:
linux modular tests work - so it is generated somewhere else. raspi is the only one which might need this atm.

Reply:
In a system that requires bundling I agree it can seem weird, but even there there are install targets for starboard, they just live within the bundles. For unbundled builds it's needed all the time (so it is needed for linux as well), and it's mapped to one explicit file in a typical install directory structure

Reply:
gotcha, leaving it as it is